<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>compress</class>
 <widget class="QWidget" name="compress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_compress">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>250</y>
     <width>95</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>苹方 中等</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton_compress{

background-color: rgb(255, 255, 255);
border-radius:10px;
border:1px solid #555;
}


QPushButton#pushButton_compress:hover{
background-color: rgb(0, 255, 127);
	color: rgb(255, 255, 255);
border-radius:10px;
border:1px solid #555;
}</string>
   </property>
   <property name="text">
    <string>压缩并保存</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>苹方 中等</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit#textEdit{
background-color: rgb(255, 255, 255);
selection-color: #0a214c; 
selection-background-color: #C19A6B;
border-radius:15px;
border:1px solid #555;
padding:5px;
}
QScrollBar:vertical
{
    width:8px;
    background:rgba(0,0,0,0%);
    margin:0px,0px,0px,0px;

}
QScrollBar::handle:vertical
{
    width:8px;
    background:rgba(0,0,0,25%);
    border-radius:4px;   
    min-height:20;
}
QScrollBar::handle:vertical:hover
{
    width:8px;
    background:rgba(0,0,0,50%); 
    border-radius:4px;
    min-height:20;
}
QScrollBar::add-line:vertical
{
    height:9px;width:8px;
    border-image:url(:/images/a/3.png);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical
{
    height:9px;width:8px;
    border-image:url(:/images/a/1.png);
    subcontrol-position:top;
}
QScrollBar::add-line:vertical:hover
{
    height:9px;width:8px;
    border-image:url(:/images/a/4.png);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical:hover
{
    height:9px;width:8px;
    border-image:url(:/images/a/2.png);
    subcontrol-position:top;
}
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{
    background:rgba(0,0,0,10%);
    border-radius:4px;
}

</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'苹方 中等'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="placeholderText">
    <string>请导入文件，再预览文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>103</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>苹方 中等</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>预览</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_input">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>85</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>苹方 中等</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton_input{

	background-color: rgb(255, 255, 255);
	border-radius:5px;
	border:1px solid #555;

}

QPushButton#pushButton_input:hover{
background-color: rgb(54, 71, 30);
	
	color: rgb(255, 255, 255);
	border-radius:5px;
	border:1px solid #555;

}

</string>
   </property>
   <property name="text">
    <string>导入文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>241</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>苹方 中等</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;
border:1px solid #555;
padding:5px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>点击左侧按钮导入文件</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>132</y>
     <width>340</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>22</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>苹方 中等</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>240</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(255, 233, 239, 255), stop:1 rgba(157, 226, 255, 255));

border-radius:20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>15</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>返回</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton_close{
border-style:inset;

border-radius:10px;
background-color: rgb(255, 255, 255);

}
QPushButton#pushButton_close:hover{
background-color: rgb(255, 0, 0);
	
	color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>pushButton_compress</zorder>
  <zorder>textEdit</zorder>
  <zorder>label_1</zorder>
  <zorder>pushButton_input</zorder>
  <zorder>lineEdit_input</zorder>
  <zorder>line</zorder>
  <zorder>label_2</zorder>
  <zorder>line_2</zorder>
  <zorder>pushButton_close</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
